src/redux/redux-state.js

```diff
@@ -0,0 +1,31 @@
+import { createStore, combineReducers } from 'redux';
+
+import profileReducer from './profileReducer';
```

Comment:
There import groups order is incorrect here.

Reply:
same above